src/protocols/secure_channel/PASESession.h

```diff
@@ -42,6 +42,10 @@
 #include <transport/raw/MessageHeader.h>
 #include <transport/raw/PeerAddress.h>
 
+namespace FuzzChip {
```

Comment:
how about using a `Testing` namespace under app? that seems more consistent than a global `FuzzChip` (at a grep for `namespace Test` yields several results).

Reply:
done